configure

```diff
@@ -1196,11 +1225,11 @@ ${ECHO_CMD} "# Generated by configure script #" >> Makefile
 ${ECHO_CMD} "#################################" >> Makefile
 ${ECHO_CMD} "ECHO_CMD = ${ECHO_CMD}" >> Makefile
 ${ECHO_CMD} "CC = ${CC}" >> Makefile
-${ECHO_CMD} "LDFLAGS += ${OSLIBS}" >> Makefile
-${ECHO_CMD} "DBLIBS += ${DBLIBS}" >> Makefile
-${ECHO_CMD} "CFLAGS += ${OSCFLAGS}" >> Makefile
+${ECHO_CMD} "LDFLAGS += ${P_LDFLAGS} ${OSLIBS}" >> Makefile
+${ECHO_CMD} "DBLIBS += ${P_DBLIBS} ${DBLIBS}" >> Makefile
+${ECHO_CMD} "CFLAGS += ${P_CFLAGS} ${OSCFLAGS}" >> Makefile
```

Comment:
What is the reason to add P_CFLAGS in both on CFLAGS and also in DBCFLAGS too?

Reply:
removed